draft-irtf-cfrg-frost.md

```diff
@@ -224,13 +236,23 @@ for more details about each.
 Beyond the core dependencies, the protocol in this document depends on the
 following helper operations:
 
+- Serializion of public key material {{dep-serialize-pk}}
 - Schnorr signatures, {{dep-schnorr}};
 - Polynomial operations, {{dep-polynomial}};
 - Encoding operations, {{dep-encoding}};
 - Signature binding {{dep-binding-factor}}, group commitment {{dep-group-commit}}, and challenge computation {{dep-sig-challenge}}
 
 This sections describes these operations in more detail.
 
+## Serialization of Public Key Material {#dep-serialize-pk}
+
+Implementations may wish to sample public key material from a distribution that either includes 
+or rejects the identity element of the underlying group. To scope this choice to only public 
+key material, we define the following two alias functions. See {{dep-pog}} for more information.
+
+- SerializePublicKey(PK): Alias for *either* SerializeElement(A) or SerializeNonIdentity(A) as required by the application.
+- DeserializePublicKey(buf): Alias for *either* DeserializeElement(buf) or DeserializeNonIdentity(buf), as required by the application. 
```

Comment:
This isn't used?

Reply:
You don't think we should define it, for symmetry?

Reply:
My take is that if it's not used it shouldn't be included, like any dead code.

Reply:
Per a conversation with @dconnolly , it is currently being used implicitly when receiving as input the group info. We agreed to make this step explicit, and to keep `DeserializePublicKey`.